packages/appium/test/e2e/cli-driver.e2e.spec.js

```diff
@@ -111,14 +111,13 @@ describe('Driver CLI', function () {
     });
 
     it('should show updates for installed drivers with --updates', async function () {
+      // Skip in CI to avoid npm registry network dependencies
```

Comment:
this comment is not the same that the one below. this test could actually be unblocked now, since we do have multiple v3 fake drivers published